nixos/modules/services/networking/wireguard.nix

```diff
@@ -255,38 +270,43 @@ let
           ${values.preSetup}
 
           ip link add dev ${name} type wireguard
+          ${lib.optionalString (!isNull values.namespace) ''
```

Comment:
In the previous conditional you have the `!` outside of the parentheses - maybe do the same here for consistency?

Reply:
Github has a 'suggestions' feature that is great for this sort of thing. Using that makes it really easy for the reviewee to apply your suggested changes.

```suggestion
          ${lib.optionalString !(isNull values.namespace) ''
```